cumulus_library/cli.py

```diff
@@ -250,7 +262,12 @@ def run_cli(args: Dict):
                     builder.clean_and_build_all(study_dict)
                 else:
                     for target in args["target"]:
-                        builder.clean_and_build_study(study_dict[target])
+                        if args["table_builder"] is not None:
```

Comment:
nit: This feels like it would read better and be more correct if you also handled the empty string case with a simple `if args["table_builder"]:`